pkg/systemd/services/timer-dialog.jsx

```diff
@@ -106,45 +164,98 @@ const CreateTimerDialogBody = ({ owner }) => {
+            }
+            // If the form base checkValidity isnt valid then check the inputs again.  this wont fail the submission unless there is an issue.  Failure will happen in the next check.
+            if (!event.target.checkValidity()) {
+                // Loop the form data keys and do some checks.  Note this ignores the repeating values.  those are captured with validation
+                const formData = new FormData(event.target);
+                for (const key of formData.keys()) {
+                    if (key == "boot-or-specific-time")
+                        continue;
+                    const input = document.querySelector('#' + key);
+                    if (input === 'undefined') {
+                        validate(event);
+                        failMessage = _("Unknown form input issue");
+                        failed = true;
+                    }
+                    if (validationFailed[key] !== 0) {
+                        input.dispatchEvent(new Event('blur'));
+                    }
+                }
+            }
+            // Confirm if any inputs are invalid
+            if (Object.values(validationFailed).includes(true))
+                failed = true;
+            // Check trigger specific validation
+            switch (delay) {
+            case 'specific-time': {
+                // Need yearly check as it doesnt have a default
+                switch (repeat) {
+                case 'yearly': {
+                    const check = repeatPatterns.find(pattern => {
+                        return !pattern.date;
+                    });
+                    if (check)
+                        failed = true;
+                    break;
+                }
+                }
+                break;
+            }
+            // Nothing to validate at this time.
+            case 'system-boot': {
+                break;
+            }
+            }
+        }
+        if (failed) {
+            setDialogError(failMessage);
+            setValidForm(false);
             return false;
+        }
 
         setInProgress(true);
 
-        // Verify if the command exists
-        const command_parts = command.split(" ");
-        cockpit.spawn(["test", "-f", command_parts[0]], { err: "ignore" })
-                .then(() => {
-                    create_timer({ name, description, command, delay, delayUnit, delayNumber, repeat, specificTime, repeatPatterns, owner })
-                            .then(Dialogs.close, exc => {
-                                setDialogError(exc.message);
-                                setInProgress(false);
-                            });
-                })
-                .catch(() => {
-                    setCommandNotFound(true);
+        create_timer({ name, description, command, delay, delayUnit, delayNumber, repeat, specificTime, repeatPatterns, owner })
+                .then(Dialogs.close, exc => {
+                    setDialogError(exc.message);
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18148-20230110-092821-56aa70de-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)